### ISSUE TITLE
Double check all default arguments in custom components in this repo

### ISSUE THREAD

**Issue by davidanthoff** [rffscghg/MimiIWG.jl#16]
https://github.com/ckingdon95/MimiIWG.jl/pull/11 fixed a bug where we assumed that `default` parameter values in a `@defcomp` would be used when that component is swapped into a model via `replace!`. With Mimi v1 that is no longer so, i.e. if there is already a model parameter that was connected to the old component, then the new component will pick up that model parameter, rather than any default value that might be set in the `@defcomp`. This new behavior strikes me as correct.

There are quite a number of components defined via `@defcomp` in this repository that have `default` values for parameters in the `@defcomp` call. We should double check all of those to make sure we don't have the same problem with those that we had with the climate sensitivity, i.e. potentially those values are not being picked up, if these components are added to models via `replace!`.

**Comment by lrennels** [comment]
@bryanparthum by the way I think it's important we double check this ... I"ll do that this week

**Comment by lrennels** [comment]
**Historically**, the above would only be the case if the model had previously been `run`, since it is on the `run` step that unconnected parameters get a default if they have one.  

**In the next Mimi Iteration** the story will be consistent regardless of runtime:  if a parameter in a model has a connected value, be that from a `default` or a call to `set!` or `update!` param, and a new component with that parameter replaces the old component ... existing parameter values  **will not change**.  If a change is desired, one can use `update_param!` to change the value, or use the version of `replace!(...; reconnect = false)` and then reconnect everything by hand.

I have looked through everything and all should be working fine, mostly because the defaults in the MimiIWG components that replace original ones have the same defaults, save for the one we fixed in #11 .  Two questions are (1) should we add explicit statements to update those parameter values even though they match and (2) is not replacing the defaults definitely the behavior we want? I think it does make sense, as it's consistent with the idea that all connections to values remain in place.

**DICE**

- _replace!(m, :co2cycle => IWG_DICE_co2cycle)_ - no defaults so no action needed
- _replace!(m, :radiativeforcing => IWG_DICE_radiativeforcing)_ -  no defaults so no action needed
- _replace!(m, :climatedynamics => IWG_DICE_climatedynamics)_ -  the new`t2xco2` has a default value of 3.0, and we handled this in #11 
- _replace!(m, :neteconomy => IWG_DICE_neteconomy)_ - no defaults so no action needed

**FUND**
- _replace!(m, :impactsealevelrise => IWG_FUND_impactsealevelrise)_ 
```
    incdens     = Parameter(default = 0.000635) # Normalization income density
    emcst       = Parameter(default = 3) # emigration loss benchmark value
    immcst      = Parameter(default = 0.4) # immigration loss benchmark
    dvydl       = Parameter(default = 1) # income density elasticity of dryland value
    wvel        = Parameter(default = 1.16) # wetland value income elasticity
    wvbm        = Parameter(default = 0.00588) # wetland value benchmark value
    slrwvpopdens0 = Parameter(default = 27.5937717888728) # wetland value income normalization value
    wvpdl       = Parameter(default = 0.47)
    wvsl        = Parameter(default = -0.11)
    dvbm        = Parameter(default = 0.004) # dryland value benchmark value
    slrwvypc0   = Parameter(default = 25000)
```
all of these match the original FUND, so should be fine, but @davidanthoff should I add explicit `set_param!` calls for consistency? 

**PAGE**

- _replace!(m, :GDP => IWG_PAGE_GDP)_
```
save_savingsrate  = Parameter(unit="%", default=15.00) #pp33 PAGE09 documentation, "savings rate".
isat0_initialimpactfxnsaturation = Parameter(unit="unitless", default=33.333333333333336) #pp34 PAGE09 documentation
```
all of these match the original PAGE, so should be fine, but @davidanthoff should I add explicit `set_param!` calls for consistency? 
- _replace!(m, :ClimateTemperature => IWG_PAGE_ClimateTemperature)_
```
frt_warminghalflife = Parameter(unit="year", default=35.00)
pole_polardifference = Parameter(unit="degreeC", default=1.50) # near 1 degC, the temperature increase difference between equator and pole
lat_g_meanlatitude = Parameter(unit="degreeLatitude", default=30.21989459076828) # Area-weighted average latitude
rlo_ratiolandocean = Parameter(unit="unitless", default=1.40) # near 1.4, the ratio between mean land and ocean temperature increases
```
all of these match the original PAGE, so should be fine, but @davidanthoff should I add explicit `set_param!` calls for consistency?

**Comment by davidanthoff** [comment]
I think if they match we can leave as is. Thanks for checking!
